website/static/js/comment.js

```diff
@@ -77,19 +261,50 @@ var BaseComment = function() {
 
     self.replying = ko.observable(false);
     self.replyContent = ko.observable('');
+    self.replyMentions = ko.observableArray([]);
+
+    self.saveContent = ko.computed(function() {
+        var content = self.replyContent();
+        if (!content) {
+            content = '';
+        }
+        self.replyMentions([]);
+        var regex = /<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/;
+        var matches = content.match(/<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/g);
```

Comment:
These two lines look really similar. Is there a way to factor out the regex that's common between them?